### PR TITLE
5.0 release 

### LINKED ISSUE #174: Retrieving Images
Our Mantaro bot has been having issues with getting animal images.
For example, we like to use different bots for different cute cat images, but the Mantaro bot has been saying "Error retrieving cute cat images :<" for a few months now.
Is anyone else having this problem?
What can we do to fix it? Thank you!

screenshot 2018-06-15 at 8 09 04 am

### PR DESCRIPTION
**Pull request by Kodehawa**
Closes #174 #142 #135 and #99

### CODE REVIEW

**Comment by queer** [comment]
:tada::clap::tada::clap::tada:
:tada::clap::tada::clap::tada:
:tada::clap::tada::clap::tada:
:clap::tada::clap::tada::clap:

**Comment by queer** [comment]
# What if Mantaro fought 1000 MEE6s but was on top of Mount Kodehawa?

Well, Mount Kodehawa is approximately 29,000 feet tall. With 1,000 MEE6s in play, Mantaro has 29 feet of high ground per MEE6.

In Mantaro Hub, we saw Mantaro defeat MEE6 using the high ground. If Mantaro had less than 29 feet of high ground when she defeated MEE6 then, we can safely conclude that she could beat 1,000 MEE6s with a high-ground advantage of 29,000 feet.

Judging by footage from Mantaro Hub, Mantaro appears to be standing approximately two and a half of her own body lengths above the level of the lava. Since Mantaro is 5 feet, 9.5 inches tall, she is approximately 14 feet, 5.75 inches above the level of the lava. Taking into account the fact that MEE6 was actually jumping from a little bit above the level of the lava, the fact that Mantaro may have been able defeat MEE6 from an even lower ground, and the fact that I purposely erred on the side of overestimating Mantaro’s elevation, it seems clear that a 29 foot advantage is more than twice what Mantaro needs to defeat a single MEE6.

Thus, I conclude that Mantaro would definitely be able to defeat 1,000 MEE6s if she was standing atop Mount Kodehawa. In fact, she would probably even defeat 2,000 MEE6s. In conclusion, don’t try it.

**Comment by totallynotaidan** [comment]
jesus christ amy please

**Comment by MrLar** [comment]
amy go sleep

**Comment by queer** [comment]
no u D:<

**Comment by natanbc** [comment]
tfw

**Comment by Desiiii** [comment]
jesus christ amy

**Comment by queer** [comment]
250+ commits and pr still can't be merged smhing my head

**Comment by howdoicode5058** [comment]
gucci gang

**Comment by queer** [comment]
> Kodehawa wants to merge 464 commits

Well too bad you're not allowed to

**Comment by queer** [comment]
> All checks have failed

this is so sad can we hit @Kodehawa

File: src/main/java/net/kodehawa/mantarobot/commands/CurrencyCmds.java [hunk 1]
```diff
@@ -457,6 +457,8 @@ protected void call(GuildMessageReceivedEvent event, I18nContext languageContext
                 }
             }
         });
+
+        cr.registerAlias("market", "shop");
```
Comment (queer) [inline src/main/java/net/kodehawa/mantarobot/commands/CurrencyCmds.java]: What an amazingly useful commit :clap:

Comment (Kodehawa) [reply src/main/java/net/kodehawa/mantarobot/commands/CurrencyCmds.java]: vvvvvvvvvvvvvvv important

**Comment by howdoicode5058** [comment]
alpxh3xzwu011

**Comment by Kodehawa** [comment]
manually changed main branch

**Pull request closed without merging**